### PR TITLE
Fix issue comment form and quick-submit

### PR DESCRIPTION
**Pull request by wxiaoguang**
1. Rewrite initGlobalEnterQuickSubmit (by the way, remove jQuery)
2. Fix issue comment form layout

### CODE REVIEW

**Comment by wxiaoguang** [comment]
https://github.com/go-gitea/gitea/pull/30623/files?diff=unified&w=1

Now everything works well.

image

File: web_src/js/features/common-global.js [hunk 1]
```diff
@@ -46,10 +46,11 @@ export function initFootLanguageMenu() {
 }
 
 export function initGlobalEnterQuickSubmit() {
-  $(document).on('keydown', '.js-quick-submit', (e) => {
-    if (((e.ctrlKey && !e.altKey) || e.metaKey) && (e.key === 'Enter')) {
+  document.addEventListener('keydown', (e) => {
+    const ieQuickSubmitEnter = ((e.ctrlKey && !e.altKey) || e.metaKey) && (e.key === 'Enter');
+    if (ieQuickSubmitEnter && e.target.matches('textarea')) {
```
Comment (delvh) [inline web_src/js/features/common-global.js]: 1. Why `ie`? What does Internet Explorer have to do with it?
2. I wonder if there will be any regression where a textarea was **explicitly** not supposed to submit.
But overall I prefer auto-submission for any textarea.

**Review by delvh — APPROVED** [review]

File: web_src/js/features/common-global.js (hunk 1 above)
Comment (wxiaoguang) [reply web_src/js/features/common-global.js]: Ooooooops .... typo ......... will fix soon

I think there should be no regression. No user would press Ctrl+Enter for any other reason at the moment.

Comment (wxiaoguang) [reply web_src/js/features/common-global.js]:
```suggestion
    const isQuickSubmitEnter = ((e.ctrlKey && !e.altKey) || e.metaKey) && (e.key === 'Enter');
    if (isQuickSubmitEnter && e.target.matches('textarea')) {
```

**Review by lunny — APPROVED** [review]

**Pull request merged**